assets/minecraft/lang/en_OC.lang

```diff
@@ -243,10 +316,9 @@ options.title=Options
 options.controls=§bControls
 options.video=§bVideo Settings
 options.language=§bLanguage
-options.stream=§bBroadcast Settings
 options.sounds=§bMusic & Sounds
-options.sounds.title=Music & Sound Options
-options.languageWarning=Language translations may not be 100% accurate
+options.sounds.title=§bMusic & Sound Options
+options.languageWarning=Language translations may not be 100%% accurate

```

Comment:
@Plastix original string:
`options.languageWarning=Language translations may not be 100%% accurate`